.github/workflows/rust.yml

```diff
@@ -1,22 +1,38 @@
 name: Rust
 
-on:
+on: 
   push:
     branches: [ "main" ]
   pull_request:
     branches: [ "main" ]
 
 env:
   CARGO_TERM_COLOR: always
+  RELEASE_TAG: 0.5.1-1 # This is the version of the package, it should be updated with every release
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
     - name: Build
       run: cargo build --verbose
```

Comment:
Why we're building a debug version and not release?

Reply:
Those are your from your original rust.yml. Would you like me to update it to build for release? So the command would be `cargo build --release --verbose`

Reply:
Oh, true. I was thinking that we're building a deb package from debug version but I didn't see that. Ok, nevermind.